newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -332,6 +336,7 @@ export const HomePage = React.memo<Props>(
           setInitialPackUserFriendlySlug,
           setInitialGameTemplateUserFriendlySlug,
           games,
+          areChaptersReady,
```

Comment:
Can you check there is no side effects on other things like game dashboard?
(small risk that the chapters are loaded and so ready, and the effect runs again even if the URL you're loading is totally not chapters related).

If it's a problem, a potential solution is to divide the effects into multiple effects with just the dependencies actually needed.